cube/src/main/java/zone/cogni/asquare/cube/operation/OperationResultProcessor.java

```diff
@@ -219,6 +218,13 @@ private SingleGroupResult processSingleFileCase(Context context, String uri, Str
     return result;
   }
 
+  private List<SingleGroupResult> processSingleFileCase(Context context, List<String> uris, String operationGroupId) {
```

Comment:
ah, I think my solution should work but maybe I missed something here. 

also, method should be moved up.